### DISCUSSION TITLE
Has the development stalled? Can we help?

### DISCUSSION THREAD

**Discussion by mquinson** [zealdocs/zeal#1407] [Q&A] [answered]
Hello there,

it seems to me that the development of Zeal stalled somehow, and I'm wondering whether there is something we could do as a community to help.

@trollixx could you please comment on your situation? If you need someone to triage the bugs, I'm voluntering the little of free time that I have, for example.

Thanks for such a great tool!
Mt

**Comment by mquinson** [comment]
Hello @trollixx, could you please let us save your work? Please comment, even if only a few words, to tell us how you imagine the future of Zeal.

Thank you so much for your time,
Mt

**Comment by trollixx** [comment]
Hey, sorry for late response. The development is stalled indeed. Not because my lack of interest in Zeal. I do hope to get back to it to a certain degree. At the very least cut the long overdue v0.7.0 release and become more responsive.

In the longer term my idea is to split the app into separate core and UI components. Over the years I've become increasingly displeased with Qt. I'd rather have native apps for each platform with shared core.

Some help with the project would be appreciated. I think handling issues, testing and fixing bugs/regressions would be the main focus areas. Adding new features would also be awesome but usually requires quite a bit of design discussion first.

**Comment by trollixx** [comment, marked as answer]
Please see the latest updates in https://github.com/zealdocs/zeal/discussions/1308.
